testsuite/performance/tests/src/main/java/org/keycloak/performance/TestConfig.java

```diff
@@ -163,7 +191,12 @@ public synchronized UserInfo next() {
 
                 String user = computeUsername(realm, idx);
                 idx += 1;
-                return new UserInfo(user, computePassword(user));
+                return new UserInfo(user, 
+                        computePassword(user),
+                        computeFirstName(idx),
```

Comment:
Firstname here uses idx incremented by 1 already and is thus out of sync with username, and email. 
One fix is to pass idx-1. Another one is to move 'idx += 1' line into a finally block around return statement.

Reply:
Good catch.